etc/schema.json

```diff
@@ -171,6 +171,12 @@
                         "items": {
                             "type": "integer"
                         }
+                    },
+                    "data": {
+                        "type": "string"
+                    },
+                    "offset": {
```

Comment:
Wonder if these fields should be grouped together in a subobject (making the meaning of offset more clear)

Reply:
It looks like a good idea. I can try implementing that to see if it works ok.